lib/api.js

```diff
@@ -73,7 +73,7 @@ function deleteHooks(reponame, url, token, callback) {
     .get(apiUrl)
     .set('Authorization', 'token ' + token)
     .set('User-Agent', 'StriderCD (http://stridercd.com)')
-    .end(function (res) {
+    .end(function (res) { //CHECK: have a callback with the first param as err and process that err
```

Comment:
`function (err, res)` is standard node callback pattern, so I think stick with that.